### PR TITLE
Remove unneeded aliases for dependent functions from package objects

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

**Comment by eparejatobes** [comment]
done for klists

**Comment by eparejatobes** [comment]
merging

File: docs/src/main/scala/cosas/ops/typeSets/Mappers.scala.md [hunk 1]
```diff
@@ -40,21 +40,21 @@ Mapping a set to a List: normally, when you are mapping everything to one type
...
-    new MapToList[F, ∅] with InContainer[X] { def apply(s: ∅): Out = Nil }
+        MaptoList[F, ∅] with InContainer[X] = 
+    new MaptoList[F, ∅] with InContainer[X] { def apply(s: ∅): Out = Nil }
   
   implicit def one[H, F <: Poly1, X]
     (implicit h: Case1.Aux[F, H, X]): 
-          MapToList[F, H :~: ∅] with InContainer[X] = 
-      new MapToList[F, H :~: ∅] with InContainer[X] { 
+          MaptoList[F, H :~: ∅] with InContainer[X] = 
+      new MaptoList[F, H :~: ∅] with InContainer[X] {
```
Comment (laughedelic) [inline docs/src/main/scala/cosas/ops/typeSets/Mappers.scala.md]: This is a bit strange. Or is it just because of `s/ToList/toList/`?

Comment (eparejatobes) [inline docs/src/main/scala/cosas/ops/typeSets/Mappers.scala.md]: :) yup. I'm going to clean the docs

**Pull request merged**